src/styles/components/markdown-editor-preview.styl

```diff
@@ -0,0 +1,2 @@
+.markdown-editor-preview
+  display none
```

Comment:
Do you think people will want to use the preview button to look at their changes? We can leave that until later but worth keeping in mind.